packages/react/src/components/Tile/Tile.js

```diff
@@ -548,6 +560,8 @@ export class ExpandableTile extends Component {
       handleClick, // eslint-disable-line
       tileCollapsedIconText, // eslint-disable-line
       tileExpandedIconText, // eslint-disable-line
+      tileCollapsedLabel, // eslint-disable-line
+      tileExpandedLabel, // eslint-disable-line
```

Comment:
```suggestion
      tileCollapsedIconText,
      tileExpandedIconText,
      tileCollapsedLabel,
      tileExpandedLabel,
```

are the eslint-disable comments required here? I actually don't think they're needed for `tileCollapsedIcontText` or `tileExpandedIconText` either since they're all referenced a few lines further down

Reply:
Doesn't seem like they are, should be good to remove them 👍